### DISCUSSION TITLE
Max button in send window is dangerous

### DISCUSSION THREAD

**Discussion by samgranger** [iotaledger/firefly#1531] [Feedback]
## Description

In the send window, the max button can easily be clicked on by mistake, ideally the user should be prompted with a message asking if they are sure that they want to send all their tokens or maybe another solution if anyone has a better proposal.

## Motivation

People click on things by mistake. This should make it safer for the end user. 

## Requirements

Write a list of what you want this feature to do.

1. Confirm if the user really wants to send the maximum available tokens.

## Open questions (optional)

Does anyone have a better proposal?

## Are you planning to do it yourself in a pull request?

No

**Comment by cvarley100** [comment]
@samgranger We do have the confirmation window. But I suppose someone could be dozing while they send a transfer. I'll ask UX their thoughts.
